cmd/milvus/milvus.go

```diff
@@ -3,339 +3,59 @@ package milvus
 import (
 	"flag"
 	"fmt"
-	"io"
-	"io/ioutil"
-	syslog "log"
 	"os"
-	"path"
-	"runtime"
-	"strings"
-	"syscall"
-
-	"github.com/gofrs/flock"
-	"github.com/milvus-io/milvus/cmd/roles"
-	"github.com/milvus-io/milvus/internal/util/paramtable"
-	"github.com/milvus-io/milvus/internal/util/typeutil"
-	"go.uber.org/automaxprocs/maxprocs"
-
-	// use auto max procs to set container CPU quota
-	"go.uber.org/zap"
-
-	"github.com/milvus-io/milvus/internal/log"
-	"github.com/milvus-io/milvus/internal/util/metricsinfo"
 )
 
 const (
-	roleMixture = "mixture"
+	DryRunCmd = "dry-run"
```

Comment:
why add a dry run, it seems does nothing

Reply:
This was previously included.